src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -20,69 +19,61 @@ import FormSection from './FormSection';
  *
  * @memberof Forms
  * @name CrossPodWriteForm
+ * @returns {React.ReactElement} The form
  */
-
 const CrossPodWriteForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
-  const { selectedUser } = useContext(SelectUserContext);
-  const [username, setUsername] = useState('');
   const [docType, setDocType] = useState('');
+  const [expiryDate, setExpiryDate] = useState('');
+  const [description, setDescription] = useState('');
+  const [file, setFile] = useState('');
```

Comment:
Similar to `UploadDocumentForm`, think `null` will be the better choice for initial state since setFile takes in a File Object from choose file.